tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -4184,22 +4204,30 @@ LogicalResult ConvertTFLArgMaxOp::matchAndRewrite(
 
 LogicalResult ConvertTFLArgMinOp::matchAndRewrite(
```

Comment:
ArgMax changes should be in a different CL.

Reply:
had to fix this here because argmax with uint8 tests broke when convert-tfl-uint8 changed.